modules/kmm-using-signing-with-kmm.adoc

```diff
@@ -0,0 +1,24 @@
+// Module included in the following assemblies:
+//
+// * hardware_enablement/kmm-kernel-module-management.adoc
+
+:_content-type: CONCEPT
+[id="kmm-using-signing-with-kmm_{context}"]
+= Using signing with Kernel Module Management (KMM)
+
+On a Secure Boot enabled system, all kernel modules (kmods) must be signed with a public/private key-pair
+enrolled into the Machine Owner's Key or MOK database. Drivers distributed as part of a distribution should already be signed by the distributions private key,
+but for kernel modules build out-of-tree, KMM supports signing kernel modules using the `sign` section of the kernel mapping.
+
+For more details on using Secure Boot see:
+
+* link:https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/9/html/managing_monitoring_and_updating_the_kernel/signing-kernel-modules-for-secure-boot_managing-monitoring-and-updating-the-kernel[Signing kernel modules for secure boot]
+* link:https://wiki.debian.org/SecureBoot[SecureBoot]
+
+.Prerequisites
+
+To use this functionality you need:
+
+* A public private key pair in the correct (der) format.
+* At least one secure-boot enabled node with the public key enrolled in its MOK database.
+* Either a pre-built driver container image, or the source code and `Dockerfile` needed to build one in-cluster.
```

Comment:
```suggestion
* A public private key pair in the correct (DER) format;
* At least one secure-boot enabled node with the public key enrolled in its MOK database;
* Either a pre-built driver container image, or the source code and `Dockerfile` needed to build one in-cluster.
```

Reply:
Changed "der" to "DER". Did not make the other changes  (;) as they contradict Red Hat style guidelines.